components/sidebar/sidebar_next_steps/sidebar_next_steps.scss

```diff
@@ -5,6 +5,11 @@
     align-self: flex-end;
     position: relative;
     background-color: var(--sidebar-bg);
+    cursor: pointer;
+
+    &:hover:not(.active) {
+        background-color: rgba(255, 255, 255, 0.08) !important;
```

Comment:
it's generally discouraged to use `!important` is there any other way to show the background? maybe a more specific selector?

Reply:
Also, is there a theme color we can use here?

Reply:
Turns out don't actually need the important. Think I just forgot to remove it. Added a variable also.